MediaBrowser.Model/Dto/BaseItemDto.cs

```diff
@@ -785,6 +785,12 @@ public class BaseItemDto : IHasProviderIds, IItemDto, IHasServerId
         /// <value>The LUFS Value.</value>
         public float? LUFS { get; set; }
 
+        /// <summary>
+        /// Gets or sets the LUFSAlbum value.
+        /// </summary>
+        /// <value>The LUFS Value.</value>
+        public float? LUFSAlbum { get; set; }
+
```

Comment:
The only issue with doing this is that then the track itself doesn't have the information to give to the web-ui. Not sure if you can get the album data in javascipt somehow then get the LUFS from that. Any ideas?

Reply:
I'm not that familiar with the web client but the data should be available when initializing playback so I guess the value should be saved somewhere at that moment.

Reply:
Probably leave this in then? Probably will change it to be AlbumLUFS as this makes more sense to read and it is then consistent with other DTO's such as AlbumArtist and AlbumId etc

Reply:
Usually this stuff is saved as a JSON blob inside the D, so whatever is added is saved without modifying the DB layer.

Reply:
> Probably leave this in then?
Not sure how you came to that conclusion. The BaseItemDto class is already bloated and a PITA to work with because all the fields are optional etc. So no, remove it and use the value from the album BaseItem instead.

> Usually this stuff is saved as a JSON blob inside the DB, so whatever is added is saved without modifying the DB layer.

I meant like saving it in a variable in jellyfin-web so it can be accessed when necessary.

Reply:
Okay, I've removed it now